### DISCUSSION TITLE
What came first, Odin II synth logo or Odin lang logo?

### DISCUSSION THREAD

**Discussion by check-m4te** [odin-lang/Odin#2718] [General]
I realised there's a free synth VST called Odin II which has the same logo as Odin, the only difference is that the synth has a line in the I so I was curious.

Odin II Synth
![image](https://github.com/odin-lang/Odin/assets/132251469/b837c81e-e8c7-4d14-8a02-cad6e5a37666)

Odin lang:
![image](https://github.com/odin-lang/Odin/assets/132251469/5809d85e-3a3f-44da-b9b8-f525a3904a13)

**Comment by check-m4te** [comment]
My guess is that the logo is just a modified font?

**Comment by leecommamichael** [comment]
[Here's a Discord link to when Bill was designing the logo.](https://discord.com/channels/568138951836172421/569272065434058776/632510758118227989)

Bill used the Nordic Free Font by Yana Bereziner. The VST name being the same is a coincidence, as Odin was just a codename for this programming language. The Odin VSTs have since moved to a difference visual design.
